4-ica_pca_feature_reconstruction.R

```diff
@@ -53,6 +51,9 @@ for (seed in filename.seeds) {
   rep.count <- grep(seed, filename.seeds)
   message(paste("\n\n#### RECONSTRUCTION ROUND", 
                 rep.count, "of", length(filename.seeds), "####\n\n"))
+
+  # set seed for ICA
+  set.seed(initial.seed)
```

Comment:
No, I mean set the user supplied (or default) `initial.seed` every time a new repeat of the pipeline is performed. One repeat takes ~25 hours on my machine. The rationale is that if something goes wrong and you have to restart the pipeline, you can pick up from the repeat you're on and get the same answer. Also, moving this seed has very minimal effects on the results of the analysis from what I've seen.